openssh/files/sshd_config

```diff
@@ -17,14 +17,33 @@ ListenAddress {{ server.bind.address }}
 
 Protocol 2
 # HostKeys for protocol version 2
+HostKey /etc/ssh/ssh_host_ed25519_key
```

Comment:
+5 lines

there should be on pillar structure defined what type of key to use.
# default ed25519 + rsa (generated and configured)

```
from mind:
openssh:
  server:
     hostkey:
        - '/etc/ssh/ssh_host_ed25519_key':
             type: ed25519
             name: ssh_host_ed25519_key             # optional
             path: /etc/ssh/                                        # optional
```
Formula should generate them if they don't exist..